index.html

```diff
@@ -103,4 +103,3 @@ <h2>Ready to get started?</h2>
         padding-top: 2.8rem;
     }
 </style>
-
```

Comment:
The only change to this file is removing an empty line. Please revert this change.